src/tls/Makefile.Kremlin

```diff
@@ -1,24 +1,27 @@
 all: build
 
-build: build-internal-test build-library build-external-test
+build: prerequisites build-internal-test build-library build-external-test
 
 test: test-internal-test # test-library test-external-test
 
-clean: clean-internal-test clean-library clean-external-test
+ifneq (,$(MITLS_USE_OPENSSL))
+CONCRETE_FLAVOR	= -OpenSSL
+endif
 
 # To include Makefile.common, you need to specify
-# the following three variables
-FLAVOR		= Kremlin
+# the following five variables
+FLAVOR		= Kremlin$(CONCRETE_FLAVOR)
```

Comment:
isn't this going to run `fstar --codegen Kremlin$(CONCRETE_FLAVOR)`?

Reply:
Darn. Good catch, thanks! Fix in progress.